### PR TITLE
 bug 1151911 - add SSL for collector+webapp

### PR DESCRIPTION
**Pull request by rhelmer**

### CODE REVIEW

File: terraform/variables.tf [hunk 1]
```diff
@@ -3,6 +3,8 @@ variable "access_key" {}
 variable "secret_key" {}
 variable "secret_bucket" {}
 variable "subnets" {}
+variable "collector_cert" {}
```
Comment (phrawzty) [inline terraform/variables.tf]: Are `collector_cert` and `webapp_cert` in the secret bucket?

File: terraform/variables.tf [hunk 2]
```diff
@@ -3,6 +3,8 @@ variable "access_key" {}
...
+variable "webapp_cert" {}
```
Comment (phrawzty) [inline terraform/variables.tf]: These files (or wildcards?) should be in `.gitignore` as well.

File: terraform/variables.tf (hunk 1 above)
Comment (rhelmer) [inline terraform/variables.tf]: They are simply names of the certs loaded in AWS, the terraform docs example is ```arn:aws:iam::123456789012:server-certificate/certName```

File: terraform/variables.tf (hunk 2 above)
Comment (rhelmer) [inline terraform/variables.tf]: @phrawzty the cert itself lives in AWS so neither the terraform user nor the instances need to have the files themselves locally.

Comment (phrawzty) [inline terraform/variables.tf]: Aaah ok that's great.

**Comment by phrawzty** [comment]
`r+`

**Pull request merged**